package.json

```diff
@@ -96,7 +96,8 @@
     "unist-builder": "^4.0.0",
     "unist-util-visit": "^5.0.0",
     "vue": "^3.3.4",
-    "vue-material-design-icons": "^5.1.2"
+    "vue-material-design-icons": "^5.1.2",
+    "vue-select": "^4.0.0-beta.6"
```

Comment:
I fell back to the upstream library here, because nextcloud/vue-select does not have a vue3 version yet. Once we migrated this, we have to change the dependency here.

Reply:
See https://github.com/nextcloud-deps/vue-select/issues/23